app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountController.scala

```diff
@@ -228,17 +228,20 @@ class BankAccountController @Inject() (
 
 object BankAccountController {
 
-  val accountNumberRegex: Predicate[String]        = "^\\d{8}$".r.pattern.asPredicate()
+  val accountNumberRegex: Predicate[String] = "^\\d+$".r.pattern.asPredicate()
+
   val accountNumberMapping: Mapping[AccountNumber] =
-    nonEmptyText(minLength = 6, maxLength = 8)
+    nonEmptyText
+      .transform[String](s => s.replaceAll("[-( )]+", ""), identity)
+      .verifying("error.length", s => s.length >= 6 && s.length <= 8)
+      .verifying("error.invalid", a => { println(accountNumberRegex.test(a)); accountNumberRegex.test(a) })
```

Comment:
do you still need `.verifying("error.invalid", a => { println(accountNumberRegex.test(a)); accountNumberRegex.test(a) })`

Reply:
I mean - println(accountNumberRegex.test(a))

Reply:
erm ... no!